### ISSUE TITLE
Game Crash on new Planet/World load w/ Mod'd Biome

### ISSUE THREAD

**Issue by jaxmacky** [zmaster587/AdvancedRocketry#1524]
## Version of Advanced Rocketry
1.12.2-1.5.0-131
## Have you verified this is an issue in the latest unstable build
yes
## Version of LibVulpes
1.12.2-0.4.0-52
## Version of Minecraft
1.12.2
## Crash report or log (if applicable)

https://paste.dimdev.org/meruhugemo.mccrash

## Description of the problem

Game crash when loading new Planet/World with mod'd biome. Tested w/ Traverse Mod & Project: Vibrant Journeys Mod. 

Deleting both Dim file and planetDefs from saves->[world-name]->advRocketry and replacing Mod'd biome with Vanilla biome  nets a loaded world without crash. 

Edit: With Vanilla biome is include inside tag w/ Mod'd biome, no crash happens, but mod's biome does not gen either,

Specific code used to generate Planet:
    `<planet name="Dunes" DIMID="20">
        <isKnown>false</isKnown>
        <fogColor>1.0,1.0,1.0</fogColor>
	<skyColor>0.8,0.7409648,0.33749646</skyColor>
	<gravitationalMultiplier>100</gravitationalMultiplier>
	<orbitalDistance>134</orbitalDistance>
	<orbitalPhi>-1</orbitalPhi>
	<rotationalPeriod>25499</rotationalPeriod>
	<atmosphereDensity>166</atmosphereDensity>
	<biomeIds>traverse:mountainous_desert</biomeIds>
	<oceanBlock>minecraft:lava</oceanBlock>
    </planet>`

**Comment by jaxmacky** [comment]
Issue was related to a improper biome id. However, without another valid biome id included in the biomeIds tag, this issue would persist.

**Comment by jaxmacky** [comment]
So after revisiting this with more time, I have had mixed results getting certain Mod Biomes to load and NOT crash the Game. Using the TellMe Mod to get Biome ID's, I used a defalt planet type and single biomes for testing. I tested both the biome id number, as well as the biome id name. Both resulted in a game crash. Using the biome id number, the planetDef.xml in the saves folder, returns the proper biome id name confirming proper naming. I deleted both the world file, and the planetDefs.xml from the saves folder each time.

I am getting game crashes from both Traverse Mod Biomes, and Project: Vibrant Journeys Biomes. Multiple biomes were tested. 

I was able to get both Stygian Mod biomes to load properly, as well as all BOP biomes,

Crash Report: https://pastebin.com/grAtAt36
